11 07, 2021 00:00:45 上午 com.zzy.diytomcat.catalina.HttpProcessor error
ERROR: this is a deliberately created exception
java.lang.RuntimeException: this is a deliberately created exception
	at com.zzy.diytomcat.servlets.DefaultServlet.service(DefaultServlet.java:37)
	at com.zzy.diytomcat.catalina.HttpProcessor.execute(HttpProcessor.java:34)
	at com.zzy.diytomcat.catalina.Connector$1.run(Connector.java:42)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
